src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -4,6 +4,37 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="event_dispatcher.aliases" type="collection">
+            <parameter key="Symfony\Component\Console\Event\ConsoleCommandEvent">console.command</parameter>
+            <parameter key="Symfony\Component\Console\Event\ConsoleErrorEvent">console.error</parameter>
+            <parameter key="Symfony\Component\Console\Event\ConsoleTerminateEvent">console.terminate</parameter>
+            <parameter key="Symfony\Component\Form\PreSubmitEvent">form.pre_submit</parameter>
+            <parameter key="Symfony\Component\Form\SubmitEvent">form.submit</parameter>
+            <parameter key="Symfony\Component\Form\PostSubmitEvent">form.post_submit</parameter>
+            <parameter key="Symfony\Component\Form\PreSetDataEvent">form.pre_set_data</parameter>
+            <parameter key="Symfony\Component\Form\PostSetDataEvent">form.post_set_data</parameter>
```

Comment:
The namespace is not right for form events (classes are in `Symfony\Component\Form\Event`).

Reply:
fixed thanks